src/app/pages/OnDemandAudioPage/OnDemandAudioPage.jsx

```diff
@@ -45,23 +48,29 @@ const getGroups = (zero, one, two, three, four, five) => ({
   group5: five,
 });
 
-const StyledGelWrapperGrid = styled(GelPageGrid)`
+const StyledGelWrapperGrid = styled.div`
+  display: grid;
+  display: -ms-grid;
+  -ms-grid-columns: 1fr 2fr;
```

Comment:
I'm not sure we need to manually include style prefixes in our source code. I think this is something Emotion takes care of when building the styles

Reply:
Thank you for the review @jroebu14. For some reason the `display` property was not showing in Internet explorer. Let me investigate other ways of retaining the GelGrid while retaining `-ms-grid`

Reply:
@jroebu14 I decided to go with custom fallbacks for this component instead